lib/console.js

```diff
@@ -139,12 +139,14 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
-Console.prototype.time = function time(label) {
+Console.prototype.time = function time(label = 'default') {
```

Comment:
This would have changed the `length` unless it was bound, which some (@jasnell) consider `semver-major`.
It does change `console.__proto__.time.length` a.k.a. `console.Console.prototype.time.length`.